internal/devconfig/packages.go

```diff
@@ -49,27 +37,33 @@ func (pkgs *Packages) VersionedNames() []string {
 
 // Get returns the package with the given versionedName
 func (pkgs *Packages) Get(versionedName string) (*Package, bool) {
-	name, version := parseVersionedName(versionedName)
-	for _, pkg := range pkgs.Collection {
-		if pkg.name == name && pkg.Version == version {
-			return &pkg, true
-		}
+	name, _ := parseVersionedName(versionedName)
+	i := pkgs.index(name)
+	if i == -1 {
```

Comment:
I didn't think that was possible so I omitted it (it's definitely impossible with the object). I assumed it would complain that the two packages conflict. I can add it back.

Reply:
Updated it to use the name and version everywhere.